resource-types/GithubRelease.dhall

```diff
@@ -0,0 +1,75 @@
+{ meta =
+  { name = "github-release", repository = "concourse/github-release-resource" }
+, Version = { Type = { version : Optional Text }, default.version = None Text }
```

Comment:
Based on https://github.com/concourse/github-release-resource/blob/3967b17bf6055a24a4dd610445df3a4e41e97019/resources.go#L86 this seems to be:

```dhall
{ id : Text
, timestamp : Optional Text
, tag : Optional Text
}
```

Reply:
Actually, based on the example https://github.com/concourse/github-release-resource#example both `id` and `timestamp` seem to actually be optional.